aviatrix/data_source_aviatrix_all_transit_gateways_test.go

```diff
@@ -0,0 +1,126 @@
+package aviatrix
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccDataSourceAviatrixAllTransitGateways_basic(t *testing.T) {
+	rName := acctest.RandString(5)
+	resourceName := "data.aviatrix_all_transit_gateways.foo"
+
+	skipAcc := os.Getenv("SKIP_DATA_ALL_TRANSIT_GATEWAY")
+	if skipAcc == "yes" {
+		t.Skip("Skipping Data Source All Transit Gateway tests as SKIP_DATA_ALL_TRANSIT_GATEWAY is set")
+	}
+	if skipAcc != "yes" {
```

Comment:
Since it is not possible for `skipAcc == "yes" && skipAcc != "yes"` to be false, this second if statement should not be required and can be removed.

Reply:
have fixed, remove this part